irohad/validation/impl/stateful_validator_impl.cpp

```diff
@@ -76,8 +135,29 @@ namespace iroha {
 
       log_->info("transactions in verified proposal: {}",
                  validated_proposal.transactions().size());
-      return std::make_shared<decltype(validated_proposal)>(
-          validated_proposal.getTransport());
+      return std::make_pair(std::make_shared<decltype(validated_proposal)>(
+                                validated_proposal.getTransport()),
+                            transactions_errors_log);
+    }
+
+    std::string StatefulValidatorImpl::formSignaturesErrorMsg(
```

Comment:
shouldn't be in the class interface, declare it as a non-class static function

Reply:
It is not (and I think not going to be) used anywhere, so maybe it's a bad idea to move it out of the class scope